css/styles.css

```diff
@@ -54,13 +87,26 @@ footer .fa {
 }
 
 .fa-github {
-    color: #000000;
+    color: var(--git-color);
```

Comment:
why does this color need a different variable?

Reply:
I think you are talking about the GitHub icon color. 
So, the GitHub icon has a black color and when we will shift to the dark mode it will not be visible. So, we need to change it to white after shifting to Dark Mode.